d3a_interface/settings_validators.py

```diff
@@ -76,6 +82,9 @@ def validate_global_settings(settings_dict):
             int(settings_dict["grid_fee_type"]) not in ConstSettings.IAASettings.VALID_FEE_TYPES:
         raise D3ASettingsException(f'Invalid value for grid_fee_type '
                                    f'({settings_dict["grid_fee_type"]}).')
+    if "slowdown" in settings_dict and settings_dict["slowdown"] > 0 and \
+            slot_length_realtime > GlobalConfig.slot_length_realtime:
+        raise D3ASettingsException("'slowdown' and 'slot_length_realtime' can not be set together")
```

Comment:
Can you confirm `slot_length_realtime ` even be greater than the actual market_slot, like `market_slot=5mins & slot_length_realtime =30mins`. Personally it would look super-wierd to me like simulation running slower than the slot_length

Reply:
IMO there should not be a limit for `slot_length_realtime` as we deprecate slowdown now. There could be the case when the user set the `slot_length` to 30 seconds, but wants to monitor it in slower speed, hence setting `slot_length_realtime` to a higher value. 
If you agree, I will now close this PR.

Reply:
One addition to that:
Of course, there is always the limit of the maximum simulation speed, meaning if the slot needs 5 mins to pass, the users setting of e.g. slot_length_realtime=1 min will not have an effect.